assets/src/design-system/components/snackbar/snackbarProvider.js

```diff
@@ -23,9 +23,9 @@ import { v4 as uuidv4 } from 'uuid';
 /**
```

Comment:
💭 I feel like the `SnackbarProvider` and the `useSnackbar` hook shouldn't live in the `components` folder. Maybe they should live in their own folder in the design system like `contexts` or something 🤔. Not a big deal, but what do you think? Am I just being silly?

Reply:
i really like the idea of separating the `SnackbarProvider` and `useSnackbar` out of components. +1 for `/contexts`.